pkg/metrics/features/metrics.go

```diff
@@ -164,7 +164,6 @@ var (
 		datapathOption.DatapathModeVeth,
 		datapathOption.DatapathModeNetkit,
 		datapathOption.DatapathModeNetkitL2,
-		datapathOption.DatapathModeLBOnly,
```

Comment:
Do we need to adjust (re-generate?) the corresponding metrics feature doc too? https://github.com/cilium/cilium/blob/pr/lb-improvements/Documentation/observability/feature-metrics-agent.txt#L311-L311